### PR TITLE
small update for unknown assets

### PR DESCRIPTION
**Pull request by birdage**
@FBRTMaka as discussed.

@ednad  @DanielJMaher this is the case for when an asset does not exist in the asset list for an A/A definition exists

### CODE REVIEW

File: ooiservices/app/main/alertsalarms.py [hunk 1]
```diff
@@ -127,54 +127,66 @@ def get_alert_alarm_status():
...
+            if aa_item['reference_designator'] not in aa_def_list:
+                #used to identify halth sensors            
+                aa_def_list.append(aa_item['reference_designator'])
+            if aa_item['reference_designator'] not in assets_names:
+                #means an asset was in the A/A definition, that was not in the asset list returned  
+                #create and add it so we can see the status, the TOC may not reflect this          
+                print "Ref-Des not in asset name list: ERROR: appending", aa_item['reference_designator']            
+
+                new_asset = {'ref_des':aa_item['reference_designator'],
+                             'coordinates': [0, 0],
```
Comment (birdage) [inline ooiservices/app/main/alertsalarms.py]: we dont have any information so we have to set it

File: ooiservices/app/main/alertsalarms.py [hunk 2]
```diff
@@ -127,54 +127,66 @@ def get_alert_alarm_status():
...
+                aa_def_list.append(aa_item['reference_designator'])
+                assets_names.append(aa_item['reference_designator'])
 
 
     #use all the info to create status
     for asset in assets_dict:
         d = asset['ref_des']
         #create inital entry   
-
         if 'manufactureInfo' in asset:
```
Comment (birdage) [inline ooiservices/app/main/alertsalarms.py]: some times manufactureInfo is not available in an asset record

**Comment by FBRTMaka** [comment]
@birdage , hey, pulled it down and checking it over now

**Comment by birdage** [comment]
@FBRTMaka ok!

**Comment by FBRTMaka** [comment]
@birdage Looks good! It's all set on your end?

**Comment by FBRTMaka** [comment]
@birdage Merging!

**Pull request merged**